pkg/apimachinery/identity/context.go

```diff
@@ -58,7 +58,7 @@ func newInternalIdentity(name string, namespace string, orgID int64) Requester {
 // This is useful for background tasks that has to communicate with unfied storage. It also returns a Requester with
 // static permissions so it can be used in legacy code paths.
 func WithServiceIdentity(ctx context.Context, orgID int64) (context.Context, Requester) {
-	r := newInternalIdentity(serviceName, "", orgID)
+	r := newInternalIdentity(serviceName, "*", orgID)
```

Comment:
Do we actually need access to all namespaces?
Would that work for you?
```suggestion
	r := newInternalIdentity(serviceName, types.OrgNamespaceFormatter(orgID), orgID)
```
Using a specific org (orgID = 1) with the wildcard namespace seems a little unusual.

Reply:
What do you mean? The `OrgNamespaceFormatter` works with orgs:

https://github.com/grafana/authlib/blob/main/types/namespace.go#L19-L24

Reply:
Oh, hm, I hadn't noticed the `if id == 1` check... I'll try this out and update the PR!